heart/heartbeat.py

```diff
@@ -471,11 +490,11 @@ def on_msg(self, ct, msg):
 				d["address"] = str(self.addr)
 				d["port"], d["players_current"], d["players_max"] = struct.unpack("<HHH", msg[:6])
 				msg = msg[6:]
-				d["name"] = stripnul(msg[:30])
+				d["name"] = msg[:30].strip("\x00")
```

Comment:
yes indeed, I noticed a discreptancy between the function name and what it did and assumed it was a bug. Can't think of a better name for it though.